src/main/java/net/mcreator/ui/workspace/WorkspacePanelVariables.java

```diff
@@ -18,6 +18,7 @@
 
 package net.mcreator.ui.workspace;

```

Comment:
Clicking dropdowns in this panel of variable type throws an exception. Check this, fix, and test.